disttools/setup.py

```diff
@@ -10,6 +10,7 @@
 class BuildParser(bdist_wheel):  # type: ignore[misc]
     def run(self) -> None:
         subprocess.run(["gprbuild", "-Plibrflxlang.gpr", "-XLIBRARY_TYPE=relocatable"], check=True)
+        super().run()
```

Comment:
Does that change have any effect?

Reply:
Yes, it runs the actual `bdist_wheel` which does all the packaging. This only "worked" by accident in the past through the fallback to the legacy install method (which we want to avoid specifically).